__tests__/components/editor/InputLiteral.test.js

```diff
@@ -172,27 +172,20 @@ describe('When the user enters input into field', ()=>{
 })
 
 describe('when there is a default literal value in the property template', () => {
+  const mockMyItemsChange = jest.fn()
+  const mockRemoveItem = jest.fn()
+
   it('sets the default values according to the property template if they exist', () => {
     plProps.propertyTemplate['valueConstraint'] = valConstraintProps
 
-    const setDefaultsForLiteralWithPayLoad = jest.fn()
-    const defaultsForLiteral = jest.fn()
-    defaultsForLiteral.mockReturnValue({
-      content: "content",
-      id: 0,
-      bnode: { termType: 'BlankNode', value: 'n3-0'},
-      propPredicate: "predicate"
-    })
-
     const wrapper = shallow(<InputLiteral {...plProps} id={12}
                           blankNodeForLiteral={{ termType: 'BlankNode', value: 'n3-0'}}
+                          handleMyItemsChange={mockMyItemsChange}
                           rtId={'resourceTemplate:bf2:Monograph:Instance'}
-                          setDefaultsForLiteralWithPayLoad={setDefaultsForLiteralWithPayLoad}
-                          defaultsForLiteral={defaultsForLiteral}
     />)
-    expect(setDefaultsForLiteralWithPayLoad).toHaveBeenCalledTimes(1)
-    expect(defaultsForLiteral).toHaveBeenCalledTimes(1)
-    expect(wrapper.instance().lastId).toEqual(0)
+    wrapper.setProps({ formData: { items: valConstraintProps.defaults } })
+    wrapper.instance().forceUpdate()
+    expect(wrapper.find('#userInput')).toBeTruthy()
```

Comment:
is this adequately testing what it is supposed to?  It doesn't seem to be checking for default values at all ...

Reply:
Changed tests so we're actually testing the value set earlier; got sidetracked in spoofing strategies with Redux store on this issue, for now the .setProps is noted as a mock attempt to trigger the right action. From my research, [#421 RDF integration tests](https://github.com/LD4P/sinopia_editor/issues/421) should be used to actually test setting defaults in the `IndexLiteral` components based on state of the Redux store.